client/my-sites/themes/themes-magic-search-card/welcome.jsx

```diff
@@ -85,6 +85,24 @@ class MagicSearchWelcome extends React.Component {
 			}
 		);
 
+		const taxonomyTranslations = {
+			feature: i18n.translate( 'Feature', {
+				context: 'Theme Showcase filter name',
+			} ),
+			layout: i18n.translate( 'Layout', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **131** times:
`translate( 'Layout' )` *ES Score: 10*


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?